bouncer/views.py

```diff
@@ -144,9 +188,25 @@ def healthcheck(request):
     return {'status': 'ok', 'version': bouncer_version}
 
 
+def _is_http_url(url):
+    return url.startswith('http://') or url.startswith('https://')
+
+
+def _pretty_url(url):
+    """
+    Return the domain name from `url` for display.
+    """
+    parsed_url = parse.urlparse(url)
```

Comment:
Good catch! Fixed.

I had a look through the `urlparse` tests and it appears that invalid IPv6 addresses are basically the only thing that `urlparse` will reject.